webapp/channels/src/components/widgets/menu/menu_items/restricted_indicator.tsx

```diff
@@ -107,27 +110,29 @@ const RestrictedIndicator = ({
                 )}
             >
                 {useModal && blocked ? (
-                    <ToggleModalButton
-                        id={`${feature}-restricted-indicator`.replaceAll('.', '_')}
-                        className='RestrictedIndicator__button'
-                        modalId={ModalIdentifiers.FEATURE_RESTRICTED_MODAL}
-                        dialogType={FeatureRestrictedModal}
-                        onClick={handleClickCallback}
-                        dialogProps={{
-                            titleAdminPreTrial,
-                            messageAdminPreTrial,
-                            titleAdminPostTrial,
-                            messageAdminPostTrial,
-                            titleEndUser,
-                            messageEndUser,
-                            customSecondaryButton: customSecondaryButtonInModal,
-                            feature,
-                            minimumPlanRequiredForFeature,
-                        }}
-                    >
-                        {icon}
-                        {ctaExtraContent}
-                    </ToggleModalButton>
+                    <span>
```

Comment:
Do we need this extra span?

Reply:
yes it is https://github.com/mattermost/mattermost/pull/24905#discussion_r1360788142